t/ui/12-needle-edit.t

```diff
@@ -225,7 +225,13 @@ sub overwrite_needle($) {
     is($driver->find_element_by_id('needleeditor_name')->get_value(), "", "needle name input clean up");
     $driver->find_element_by_id('needleeditor_name')->send_keys($needlename);
     is($driver->find_element_by_id('needleeditor_name')->get_value(), "$needlename", "new needle name inputed");
-    $driver->find_element_by_id('save')->click();
+    if ($ENV{SELENIUM_CHROME}) {
+        $driver->find_element_by_id('save')->click();
+    }
+    else {
+        # workaround for PhantomJS
```

Comment:
Yeah. Better use 'PhantomJS sucks balls and breaks all the time, but as some developers refuse to install chromedriver we have to keep it running, so better add evil crap workarounds just because it happened to be broken in Tumbleweed yet again'